docs/_policies/workgroups.md

```diff
@@ -1,29 +1,22 @@
 # Workgroup Management
-## What is a Workgroup?
-
-The [Stanford Workgroup Manager](https://workgroup.stanford.edu/) is a web interface that gives Stanford community members a place to define a group of other community members based on their SUNet IDs for use in various applications. A workgroup is given a name that uniquely identifies it and succintly describes its purpose. Relevant to the Yens, the Workgroup Manager helps control access to individual [project space](/services/newProject.html) on ZFS (ex: `gsb-rc:[faculty-SUNet]-[projectname]`).
 
-## Adding/Removing Users to your Workgroup
-If you are an Administrator of a workgroup, you can use the Workgroup Manager interface to easily add users to and remove users from that workgroup. Adding users to a workgroup immediately grants them permissions to whatever resource that workgroup controls access to. Likewise, removing users revokes those permissions. You might want to do this if you no longer want someone to have access to your project space or as part of regular auditing of external collaborators you have sponsored access to the Yens for.
+## What is a Workgroup?
 
-Users can be added as Members and/or Administrators of a workgroup. While Members are simply granted permission to the resource controlled by the workgroup, Administrators have the power to add *other* users as Members/Administrators to the workgroup.
+The [Stanford Workgroup Manager](https://workgroup.stanford.edu/){:target="_blank"}, is a web interface that allows Stanford community members to define groups of other community members based on their SUNet IDs for use in various applications. Each workgroup is assigned a unique name that succintly describes its purpose. 
```

Comment:
Remove `,`

Reply:
Maybe reword this to not lead with the workgroup manager.  Also -- spelling error with "succinctly", and stray comma.

> A *workgroup* is a list of members in a group, identified by their SUNet IDs, and given a name that uniquely identifies it, for use on the Yen servers and at Stanford more broadly. The [Stanford Workgroup Manager](https://workgroup.stanford.edu/){:target="_blank"} is a web interface that allows Stanford community members to manage workgroup membership.

Reply:
`succintly` is still misspelled